### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by carloscds**
Include message about update application.

### CODE REVIEW

File: Documentation/dotnet-mage/README.md [hunk 1]
```diff
@@ -56,3 +56,5 @@ Suppose that we have copied project output to a sub-folder `files` and our .NET
 * Create deployment manifest
 
 `mage.net -new Deployment -Install true -pub "My Publisher" -v 1.0.0.1 -AppManifest files\MyApp.manifest -t MyApp.application`
+
+### To update an existing application, repeat steps above.
```
Comment (NikolaMilosavljevic) [inline Documentation/dotnet-mage/README.md:60]: Actually, the steps don't have to be repeated - there is an "-update" command that should be used. This is explained in the document I linked above: https://docs.microsoft.com/en-us/dotnet/framework/tools/mage-exe-manifest-generation-and-editing-tool

I suggest updating the comment to direct users to that document for more usage scenarios, like preparing an update or signing the application.

**Review by NikolaMilosavljevic — CHANGES_REQUESTED** [review]

File: Documentation/dotnet-mage/README.md (hunk 1 above)
Comment (carloscds) [reply Documentation/dotnet-mage/README.md:60]: @NikolaMilosavljevic Thanks, I will update.

File: Documentation/dotnet-mage/README.md [hunk 2]
```diff
@@ -56,3 +56,7 @@ Suppose that we have copied project output to a sub-folder `files` and our .NET
 * Create deployment manifest
 
 `mage.net -new Deployment -Install true -pub "My Publisher" -v 1.0.0.1 -AppManifest files\MyApp.manifest -t MyApp.application`
+
+### Update an existing application
+
+`mage.net -update MyApp.Application -v 1.0.0.2 -AppManifest files\MyApp.manifest -t MyApp.application`
```
Comment (NikolaMilosavljevic) [inline Documentation/dotnet-mage/README.md:62]: This line would update deployment manifest, but not the application manifest.

Here's what I propose for content of the new section "Update an existing application":

* Update application manifest

`mage.net -update files\MyApp.manifest -v 1.0.0.2

* Update deployment manifest

`mage.net -update MyApp.application -AppManifest files\MyApp.manifest -v 1.0.0.2

Comment (carloscds) [reply Documentation/dotnet-mage/README.md:68]: Thanks Nikola, updated!

Comment (NikolaMilosavljevic) [reply Documentation/dotnet-mage/README.md:68]: > Thanks Nikola, updated!

Looks good - a small ask - in second command '-t' option is not needed during update, as the application manifest is already specified, so the command would look like:

`mage.net -update MyApp.Application -v 1.0.0.2 -AppManifest files\MyApp.manifest`

Comment (carloscds) [reply Documentation/dotnet-mage/README.md]: @NikolaMilosavljevic Done! Thanks!

**Review by NikolaMilosavljevic — APPROVED** [review]

**Comment by NikolaMilosavljevic** [comment]
Thanks @carloscds - merging this PR!

**Pull request merged**